numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -2801,6 +2801,11 @@ def add_newdoc(place, name, doc):
 
     The `sign` function returns ``-1 if x < 0, 0 if x==0, 1 if x > 0``.
 
+    For complex inputs, the `sign` function returns:
+    ``-1+0j if x.real < 0,
```

Comment:
How about
```
``sign(x.real) + 0j if x.real != 0, sign(x.imag) + 0j if x.real == 0``
```
Which is somewhat shorter. Hmm, I really think sign should be returning a real here...